src/ResourceManager/Network/Commands.Network/help/Get-AzureRmRouteFilter.md

```diff
@@ -7,11 +7,14 @@ schema: 2.0.0
 
 # Get-AzureRmRouteFilter
 
-## SYNOPSIS
```

Comment:
Please add back the synopsis section with a synopsis, as the doc generation will fail without it.  Apply to all files.